src/components/EditorWrapper.vue

```diff
@@ -755,4 +758,14 @@ export default {
 	#files-public-content {
 		height: 100%;
 	}
+
+	@media only screen and (max-width: 600px) {
+		.connection-error {
+			visibility: hidden;
+		}
+		.connection-error:after {
+			visibility: visible;
+			content: 'Offline';
```

Comment:
That won't work with translations unfortunately, but you can just return the translated text of Offline within the lastSavedStatus method by using the isMobile mixin https://github.com/nextcloud/text/blob/5eb7b205b3aed1f1e6fd5bbec41c947df46d4f39/src/mixins/isMobile.js#L23